cmd/simulator/main.go

```diff
@@ -102,6 +116,25 @@ func NewSingleServer() (*server.Config, *server.Server, server.CleanupFunc) {
 		log.Fatalf("initialize logger error: %s\n", err)
 	}
 
+	lcfg := &zap.Config{
```

Comment:
I'm not sure if this is a proper way to mute the raft info log of etcd.

Reply:
I think you can pass empty `OutputPaths` and `ErrorOutputPaths` if you want to disable all logs produced by raft. Please refer to https://godoc.org/go.uber.org/zap#Config and https://godoc.org/go.uber.org/zap#Open

BTW, I suppose you want to do it in main() instead reset it before running every case.

Reply:
There is only one PD server. I think to disable the raft log is ok.